docs/view_components/welcome.rst

```diff
@@ -37,14 +37,17 @@ Requirement
 2. If the user ticks the check-box *Do not show again*, the welcome view-component should not appear to the user during any subsequent launch of the tool
 3. The user could later re-enable the appearance of the pop-up window through *Display* or *Preferences* menu in *Settings*
 
+Link with other views .. isn't this missing here?
```

Comment:
How about 

```suggestion
Link with other views 
^^^^^^^^^^^^^^^^^^^^^

This window is displayed on top of the first step of the pregression bar every time the tool is launched.
```

Reply:
this was a meta-comment for us @TheOneAndra

Reply:
yes but this is a proposition to avoid leaving the category completely empty

Reply:
@Bachibouzouk okay for you? If yes, I would merge.

Reply:
I would remain more general: "On the very first use of the tool, this view-component should be the first thing the user sees". Here the other description implies that the progression bar index is at the first step when we launch the tool and say that it wlll *always* be on top of it, which is not necessarily true: if user ticked do not show again, the welcome view-component will not show again next time we start the tool.

Reply:
> yes but this is a proposition to avoid leaving the category completely empty

Sorry I had somehow not noticed the multiline suggestion, I need to get use to those :)

Reply:
I am not sure I get your comment PF but I can propose that:

This window is displayed on top of the first step of the progression bar every time the tool is launched. If the `Do not show again` check-box has been ticked, this window will not be displayed anymore.

Reply:
I don't think it was decided that when we start the tool we are in the first step of the progression bar, the progression bar is there when we start to create an energy cell. My comment was in that direction, that the way you describe it fits your own mental picture of the tool. Which is not exactly the same as the one of Berit and Frank for example. (they don't have progression bar at first)